### PR TITLE
Silent server setup notifi. + custom hint dismiss tip local.

### PR DESCRIPTION
**Pull request by igorkis-scrts**
## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Server setup notifications produce annyoing looped "tick" sound.
Also, this PR adds localizable strings for custom hint dismiss tips.

### Please specify which Issue this PR Resolves.
-

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
-

### CODE REVIEW

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -5095,5 +5095,15 @@
...
       </Key>
     </Container>
+    <Container name="A3A_customHint">
+      <Key ID="STR_A3A_customHint_dismiss_no_key">
+          <Original>&lt;br/&gt;&lt;t size='0.8' color='#e5b348' shadow='1' shadowColor='#000000' valign='top'&gt;%1 Previous Hints&lt;br/&gt;Bind dismiss key by: &lt;t color='#f0d498'&gt;Escape Menu&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;CONFIGURE&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;CONTROLS&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;SHOW: Antistasi Plus&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;Dismiss Previous Hint&lt;/t&gt;&lt;/t&gt;</Original>
+          <Russian>&lt;br/&gt;&lt;t size='0.8' color='#e5b348' shadow='1' shadowColor='#000000' valign='top'&gt;%1 предыдущих подсказок&lt;br/&gt;Назначить клавишу закрытия подсказок: &lt;t color='#f0d498'&gt;Меню паузы&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;НАСТРОЙКА&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;УПРАВЛЕНИЕ&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;ФИЛЬТР: Antistasi Plus&lt;/t&gt; &gt; &lt;t color='#f0d498'&gt;Закрыть предыдущую подсказку&lt;/t&gt;&lt;/t&gt;</Russian>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:5101]: Says "Antistasi Plus".

**Review by jaj22 — CHANGES_REQUESTED** [review]
Otherwise looks fine.

File: A3A/addons/core/Stringtable.xml (hunk 1 above)
Comment (igorkis-scrts) [reply A3A/addons/core/Stringtable.xml:5101]: whoops

**Comment by Bob-Murphy** [comment]
@jaj22 I guess this is good now?

**Comment by jaj22** [comment]
@Bob-Murphy Actually had second thoughts. Do we really want epic formatting strings in the stringtable? Seems like that might be a pain to work with.

Alternative is to split into seven mini-strings.

**Comment by jaj22** [comment]
Your funeral. Merge it then :P

**Review by Bob-Murphy — APPROVED** [review]
lgtm

**Comment by Bob-Murphy** [comment]
@jaj22 please approve, your change request is blocking

**Review by jaj22 — APPROVED** [review]

**Comment by igorkis-scrts** [comment]
@Bob-Murphy 
Fixed.

**Pull request merged**